_posts/2016-01-21-Rust-1.6.md

```diff
@@ -0,0 +1,78 @@
+---
+layout: post
+title: "Announcing Rust 1.6"
+author: The Rust Core Team
+---
+
+Hello 2016! We’re happy to announce the first Rust release of the year, 1.6.
+Rust is a systems programming language focused on safety, speed, and
+concurrency.
+
+As always, you can [install Rust 1.6][install] from the appropriate page on our
+website, and check out the [detailed release notes for 1.6][notes] on GitHub.
+About 1100 patches were landed in this release.
+
+[install]: http://www.rust-lang.org/install.html
+[notes]: https://github.com/rust-lang/rust/blob/stable/RELEASES.md#version-160-2016-01-21
+
+### What's in 1.6 stable
+
+This release contains a number of small refinements, one major feature, and
+a change to [Crates.io](https://crates.io).
+
+#### libcore stabilization
+
+The largest new feature in 1.6 is that [`libcore`] is now stable! Rust’s standard
+library is two-tiered: `libcore` contains a lot of functionality, but more important
+is what it does _not_ contain: significant dependencies. `libcore` is completely
+platform agnostic, and requires only a handful of external symbols to be defined.
+Rust’s `libstd` builds on top of `libcore`, adding support for memory allocation,
+I/O, and concurrency. Applications using Rust in the embedded space, as well as
+those writing operating systems, often eschew `libstd`, using only `libcore`.
+
+[`libcore`]: http://doc.rust-lang.org/nightly/core/
+
+`libcore` being stabilized is a major step towards being able to write the lowest
+levels of software using stable Rust. There’s still future work to be done, however,
```

Comment:
Let's be a bit more specific about this: we can say that this step is enough to build a stable library ecosystem around `libcore`, but a bit more work is needed before stable Rust supports *applications* at the `libcore` level.